src/mudlet.pro

```diff
@@ -147,7 +147,7 @@ TEMPLATE = app
 ########################## Version and Build setting ###########################
 # Set the current Mudlet Version, unfortunately the Qt documentation suggests
 # that only a #.#.# form without any other alphanumberic suffixes is required:
-VERSION = 3.22.1
+VERSION = 3.0
```

Comment:
In the words of John McEnroe - "[You cannot be serious!](https://en.wikipedia.org/wiki/John_McEnroe#1979%E2%80%9383)" - or is this some sort of test?